packages/web-pkg/src/components/SideBar/FileSideBar.vue

```diff
@@ -383,13 +383,14 @@ export default defineComponent({
 
         // shared resources look different, hence we need to fetch the actual resource here
         try {
-          let fullResource = await clientService.webdav.getFileInfo(props.space, {
+          const webDavResource = await clientService.webdav.getFileInfo(props.space, {
             path: resource.path
           })
 
-          // make sure props from the share (=resource) are available on the full resource as well
-          fullResource = { ...fullResource, ...resource }
-          loadedResource.value = fullResource
+          // make sure props from the share (=resource) are available on the merged resource
+          const mergedResource = { ...webDavResource, ...resource }
+          mergedResource.tags = webDavResource.tags // tags don't exist in Graph API yet, hence take them from webdav
```

Comment:
I don't get it... why are the tags not present from spreading the webdav resource and the share resource into a new object? 🙈

Reply:
Because tags on a share resource (= a resource from the Graph API) are always `[]`. So it overwrites any potential tags from the `webDavResource`.

It's quite messy that we need to merge resources from both APIs with different data here :( But I guess that's the easiest "fix" until we fully switch to Graph.

Reply:
IHMO ...resource is overwriting the field, no ?

Reply:
@JammingBen did you try with lodash merge, IDK but it could be potentially smarter ... ?

Reply:
Ah, I didn't know that the graph resource also has the `tags` field with an empty array... maybe you can add that  to the comment. :-)